src/inc/gcinfotypes.h

```diff
@@ -285,10 +283,10 @@ struct GcStackSlot
 #define SECURITY_OBJECT_STACK_SLOT_ENCBASE 6
 #define GS_COOKIE_STACK_SLOT_ENCBASE 6
 #define CODE_LENGTH_ENCBASE 8
-#define STACK_BASE_REGISTER_ENCBASE 3
+#define STACK_BASE_REGISTER_ENCBASE 1 // FP encoded as 0
 #define SIZE_OF_STACK_AREA_ENCBASE 3
 #define SIZE_OF_EDIT_AND_CONTINUE_PRESERVED_AREA_ENCBASE 4
-#define NUM_REGISTERS_ENCBASE 2
+#define NUM_REGISTERS_ENCBASE 3
```

Comment:
why is this modified to be 3? Comment in related section for arm says that this is only related to perf and not correctness